runtests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh -l
+
+check_return_code () {
+   if [ $? -eq 0 ]; then
+     echo PASS
+   else
+     echo FAIL
+     exit 1
+   fi
+}
+
+if [ ! -f $JSON_PATH ]; then
+    echo "Existing Json file not found in $JSON_PATH"
+    truncate -s 0 $JSON_PATH
+fi
+
+skopeo inspect docker://$1 > $JSON_PATH
+
+check_return_code
+
+if [ -f "${JSON_PATH}" ] ; then
+    echo "JSON file Generated Successfully using $1";
+else
+    echo "JSON file is not valid";
```

Comment:
```suggestion
echo "JSON file $JSON_PATH doesn't exist";
```

Reply:
I think this is not required i wanted to validate json here.